chunky/src/java/se/llbit/chunky/ui/controller/ResourcePackChooserController.java

```diff
@@ -331,6 +325,7 @@ public void initialize(URL location, ResourceBundle resources) {
 
     Icons.buildIcon(Icons.HEAVY_PLUS).withSize(14).setAsGraphicOn(addNewTargetPackBtn);
     addNewTargetPackBtn.setOnAction(evt -> browseForUnlistedPack());
+    disableDefaultTexturesBtn.setTooltip(new Tooltip("Disable loading of textures from Minecraft and revert to internal textures and any loaded resource packs. \nRequires restart for changes to take effect."));
```

Comment:
```suggestion
    disableDefaultTexturesBtn.setTooltip(new Tooltip("Disable loading of textures from Minecraft and revert to internal textures and any loaded resource packs.\nRequires restart for changes to take effect."));
```

Reply:
Ah, thank you. I did not test it. I'm glad that you caught it.